libs/fn/src/lib/button/button.component.ts

```diff
@@ -28,70 +29,116 @@ export type ButtonType = '' | 'secondary' | 'flat' | 'link' | 'outline' | 'naked
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush,
     host: {
+        '[class]': `'fn-button ' + (class || '')`,
         '[attr.type]': 'type',
-        '[attr.disabled]': '_disabled || null'
-    }
+        '[class.is-disabled]': 'disabled',
+        '[class.fn-button--emphasized]': 'emphasized',
```

Comment:
Maybe move it to HostBinding decorators for each attribute if possible?

Reply:
They will be spread accross codebase. Initially I thought that too but, then decided to go this way